client/README.md

```diff
@@ -0,0 +1,36 @@
+<img src="images/gobra.png" height="250">
+
+Gobra IDE directly integrates verification with Gobra into VSCode.
+
+## First Usage
+1. Install the [requirements](#requirements)
+2. Install [this extension](https://marketplace.visualstudio.com/items?itemName=viper-admin.gobra-ide) from the Visual Studio Marketplace
+3. Open a Gobra or Go file to activate this extension. At first activation, this extension will automatically download Gobra.
```

Comment:
The "At first activation" is easy to confuse with "At first," in initial position. Adding a "the" might be worth it "At the first activation" to make it more distinct compared to the comman sentence introduction. I would put the "automatically" at the end of the sentence.